packages/animations/example/lib/container_transition.dart

```diff
@@ -117,18 +122,63 @@ class _OpenContainerTransformDemoState
       body: ListView(
         padding: const EdgeInsets.all(8.0),
         children: <Widget>[
+          Container(
+            key: counterKey,
+            padding: const EdgeInsets.symmetric(
+              vertical: 15.0,
+              horizontal: 10.0,
+            ),
+            margin: const EdgeInsets.only(bottom: 10.0),
+            decoration: BoxDecoration(
+              color: Colors.purple[100],
+              borderRadius: BorderRadius.circular(5.0),
+            ),
+            child: Row(
+              children: <Widget>[
+                const Padding(
+                  padding: EdgeInsets.only(right: 10.0),
+                  child: Icon(Icons.done),
+                ),
+                RichText(
+                  text: TextSpan(
+                    style: const TextStyle(fontSize: 18.0, color: Colors.black),
+                    children: <TextSpan>[
+                      const TextSpan(text: 'Marked as done '),
+                      TextSpan(
+                        text: '$doneCounter',
+                        style: const TextStyle(fontWeight: FontWeight.bold),
+                      ),
+                      const TextSpan(text: ' times'),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
           _OpenContainerWrapper(
             transitionType: _transitionType,
             closedBuilder: (BuildContext _, VoidCallback openContainer) {
               return _ExampleCard(openContainer: openContainer);
             },
+            onClosed: (bool isMarkedAsDone) {
```

Comment:
Since the methods are all the same: create an instance method for this and just pass it to all the onClosed properties.